pkg/mcoproducer/testdata/pull-driver-test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I think we should delete those files (this and the next one) and instead use the templates script written.